neps/nep-0330.md

```diff
@@ -100,10 +120,18 @@ If someone were to call the view function `contract_metadata`, the contract woul
             standard: "nep177", 
             version: "2.0.0"
         }
-    ]
+    ],
+    build_info: {
+	      build_environment_ref: "sourcescan/cargo-near@sha256:bf488476d9c4e49e36862bbdef2c595f88d34a295fd551cc65dc291553849471",
+        source_code_snapshot: "git+https://github.com/near-examples/nft-tutorial.git#39f2d2646f2f60e18ab53337501370dc02a5661c",
+	      contract_path: "nft-contract",
+	      build_command: ["cargo", "near", "deploy", "--no-abi"]
+	  }
```

Comment:
Fix the indentation 
```suggestion
    build_info: {
        build_environment_ref: "sourcescan/cargo-near@sha256:bf488476d9c4e49e36862bbdef2c595f88d34a295fd551cc65dc291553849471",
        source_code_snapshot: "git+https://github.com/near-examples/nft-tutorial.git#39f2d2646f2f60e18ab53337501370dc02a5661c",
        contract_path: "nft-contract",
        build_command: ["cargo", "near", "deploy", "--no-abi"]
    }
```

Reply:
I like flatting the structure! 👌
The `_ref` suffix is a bit out of the overall naming convention, could it be like a full word `_reference`, or maybe drop it, e.g. `build_environment`? Or `build_environment_link` (cause we already use the `link` word in a different place)?

Reply:
let's drop it.

Reply:
dropped `_ref` in aa84739feadf6b04bbfd7fdd06102b6d189367c8